cognito-user-pools/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,16 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Amazon Cognito Identity Provider"
+metadata:
+  keywords:
+    - "cognitoidentityprovider"
+    - "cognito-identity-provider"
+    - "cognito-idp"
+    - "aws"
```

Comment:
Same thing for keywords. Shouldn't they include "user pools"?

Reply:
Added "congito-user-pools"